_posts/2021-01-22-logarithmic-time-factoring-magic-trick.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Factoring in logarithmic time: a mathemagic trick"
+categories:
+author: Robin Jadoul
+meta: "Factoring in logarithmic time: a mathemagic trick"
+tags:
+excerpt_separator: <!--more-->
+---
+
+When you have an interest in cryptography, and you occasionally browse reddit, you've probably seen something like this before. Someone creates a new post on r/crypto or r/cryptography, claiming to be able to efficiently factor integers or RSA moduli. This then usually goes hand in hand with a serious lack of context and explanation, notation that is questionable at best, often just plain unreadable, and an aversion of providing proper proof by factoring an actual, cryptographically big modulus. Yet because it's hard to understand, and somethings there's math involved that at first glance seems like it might have some deeper meaning, you're left wondering, is there some insight to be found here.
+
+**Spoiler alert**: nope...
+<!--more-->
+
+## Finding the good stuff
+
+As a case study, we'll have a closer look at [this reddit post](https://www.reddit.com/r/cryptography/comments/l1zf2y/is_this_known_as_an_attack_on_rsa_what_are_the/), which asks the question if this is a known attack, and then enigmatically just links to a post on [linkedin](https://www.linkedin.com/feed/update/urn:li:ugcPost:6757583973079117824/). To get around my lack of linkedin account, someone was kind enough to send me the following image of the main content.
+
+![Image containing mostly unreadable math with a small RSA modulus factored by hand as example](https://cdn.discordapp.com/attachments/695239931790098434/801892927587876864/Screenshot_2021-01-21_at_7.16.05_pm.png)
```

Comment:
As for anonymity, can you perhaps remove the linkedin link (will probably go invalid anyway), and remove the contact details from the bottom of the image - then put it in assets/images and you can refer to it as ![](/assets/images/proof.png)

Reply:
Hmm, tried to amend + force push, but it's still here in the discussion history

Reply:
The commit still lingers somewhere on github, and I don't know if there's even a way to get it gc'd, but at least few people would come looking here, for this, anyway